src/Engine/ProtoCore/Utils/ArrayUtils.cs

```diff
@@ -428,7 +428,6 @@ public static bool GetFirstNonArrayStackValue(StackValue svArray, ref StackValue
         /// <returns></returns>
         public static int GetElementSize(StackValue array, RuntimeCore runtimeCore)
         {
-            Validity.Assert(array.IsArray);
```

Comment:
Shouldn't caller of this method ensure that this method is called on correct StackValue? I understand it may be difficult to ensure that all callers have validated the data before calling it.